styles/index.css

```diff
@@ -56,13 +56,24 @@ h6 {
 
 /* SURVEY FORM STYLES */
 
+body {
+  background-image: url(A2.jpg);
```

Comment:
Best practice:  while this seems to work, you always want to make sure you put the value in parenthesis in quotes i.e url('A2.jpg');